profiles/paul-ford/index.md

```diff
@@ -9,4 +9,4 @@ twitter_username: blank
 linkedin_username: blank
 ---
 
-In his role, Paul Ford is responsible for driving developer adoption of Chia, and Chia’s remarkable built in programming language, Chia Lisp.  Previous to Chia, Mr. Ford was CMO of crypto data storage company Storj, responsible for both product and marketing, helping drive its market value to almost $1BN USD.  Before Storj, he was VP of SendGrid ($1.5BN IPO, 2017) and then Twilio SendGrid ($3BN Acquisition, 2018).  Paul also saw great success as VP of API-based datacenter company SoftLayer in Dallas TX ($2.5BN Acquisition by IBM). Other engagements included time in Corporate Developement at Rackspace (RAX) and 2X as a venture funded startup co-founder.
\ No newline at end of file
+In his role, Paul Ford is responsible for driving developer adoption of Chia, and Chia’s remarkable built-in programming language, ChiaLisp. Previous to Chia, Mr. Ford acted as CMO of Storj, a crypto data storage company, where he was responsible for both product and marketing, helping drive its market value to almost $1BN USD. Before Storj, he was VP of SendGrid ($1.5BN IPO, 2017) and then Twilio SendGrid ($3BN Acquisition, 2018). Paul also saw great success as VP of API-based datacenter company SoftLayer in Dallas, TX ($2.5BN Acquisition by IBM). Other engagements include Corporate Developement at Rackspace (RAX), and 2X as a VC-funded startup co-founder.
```

Comment:
It is "Chialisp" "acted as" feels off but agree that something better can go there.